README.md

```diff
@@ -42,3 +42,12 @@ Now submit the pull request, selecting `objektivapp/base_expo_rn_repo` as the ba
 # More Questions?
 
 Please reach, we're happy to help!
+
+# Extra
+I have tried to create static view of the first screen, and I have not added any logics yet,I have created the Accordion component from scratch using animated component.
```

Comment:
I see this has the word "yet". Also, screenshot 2 has not been fully implemented. The body of the "Action" accordion is missing. Is this submission finished or do you plan to continue working on it? We will only review it once.

Reply:
I have completed both the screens with logics , you can check this now :)